app/views/courses/show.html.haml

```diff
@@ -41,8 +38,8 @@
           .col.s12.center
             %a.hoverable#up{ href: course_lesson_path(@course, lesson, lesson_no: "up"), "data-method" => "patch", class: "up" }
               %i.material-icons expand_less
-            %li.card-panel= link_to "Lesson #{lesson.lesson_no}: #{lesson.title}", course_lesson_path(@course, lesson)
-            %a.hoverable.highlight#down{ href: course_lesson_path(@course, lesson, lesson_no: "down"), "data-method" => "patch", class: "down" }
+            %h5.card-panel.hoverable= link_to "#{lesson.lesson_no}. #{lesson.title}", course_lesson_path(@course, lesson)
+            %a.hoverable#down{ href: course_lesson_path(@course, lesson, lesson_no: "down"), "data-method" => "patch", class: "down" }
```

Comment:
Avoid defining `class` in attributes hash for static class names<br>Line is too long. [134/80]